packages/firestore/test/integration/browser/webchannel.test.ts

```diff
@@ -67,6 +68,13 @@ describeFn('WebChannel', () => {
     // different database ID.
     stream.onMessage(msg => {
       if (msg.targetChange) {
+        // Assertion will fail when additional targets are added. This works so
+        // long as backend target id counts up from 1. We expect the second
+        // target to be a bad payload. If this does not fail stream, then we
+        // might receive message that it was successfully added. The following
+        // assertion will catch failure to validate data on backend.
+        expect(msg.targetChange?.targetIds).to.not.include(2);
```

Comment:
I'm curious
Does this test fail on the emulator without this `expect`?
Does this test fail on the emulator with this `expect`?

Reply:
👍